Tasks/ArchiveFilesV2/task.json

```diff
@@ -19,7 +19,7 @@
     "minimumAgentVersion": "2.182.1",
     "version": {
         "Major": 2,
-        "Minor": 213,
+        "Minor": 215,
```

Comment:
I don't think we need to update the task version if there are only changes in the tests